### PR TITLE
Bug 1763169 - Support production/testing mozci environments in ClassificationLoader regex

### PR DESCRIPTION
**Pull request by EvaBardou**

### CODE REVIEW

File: tests/etl/test_classification_loader.py [hunk 1]
```diff
@@ -126,22 +126,45 @@ def populate_bugscache():
...
+        ('production', 'completely bad route'),
+        ('production', 'index.project.mozci.classification..revision.A35mWTRuQmyj88yMnIF0fA'),
+        ('production', 'index.project.mozci.classification.autoland.revision.'),
+        (
+            'production',
+            'index.project.mozci.classification.autoland.revision.-35mW@RuQ__j88yénIF0f-',
+        ),
+        (
+            'production',
+            'index.project.mozci.testing.classification.autoland.revision.A35mWTRuQmyj88yMnIF0fA',
```
Comment (jmaher) [inline tests/etl/test_classification_loader.py:140]: this has `index.project.mozci.testing. ...`, why is this `mode=production`?

File: tests/etl/test_classification_loader.py [hunk 2]
```diff
@@ -126,22 +126,45 @@ def populate_bugscache():
...
+        ),
+        ('testing', 'index.project.mozci.classification.autoland.revision.A35mWTRuQmyj88yMnIF0fA'),
```
Comment (jmaher) [inline tests/etl/test_classification_loader.py:142]: should this be `production` instead of `testing`?

**Review by jmaher — CHANGES_REQUESTED** [review]
thanks for the tests, it looks like a mixup on one of the tests.

File: tests/etl/test_classification_loader.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (EvaBardou) [reply tests/etl/test_classification_loader.py:142]: Same here!

**Review by jmaher — APPROVED** [review]
ah, I missed the obvious, thanks!

**Pull request merged**